tests/ledger.rs

```diff
@@ -679,6 +679,96 @@ mod high_cases {
             TestUtils::cleanup_storage();
         }
     }
+
+    mod get_txn_requests {
+        use super::*;
+
+        #[test]
+        fn sovrin_build_get_txn_request() {
+            let identifier = "identifier";
+            let data = 1;
+
+            let expected_result = r#""identifier":"identifier","operation":{"type":"106","data":1}"#;
+
+            let get_txn_request = LedgerUtils::build_get_txn_request(identifier, data).unwrap();
+            assert!(get_txn_request.contains(expected_result));
+        }
+
+        #[test]
+        #[ignore]
```

Comment:
Add comment about ignore (also possible create ticket in JIRA) - for this and the next one test.